theano/tensor/nnet/conv3d2d.py

```diff
@@ -198,8 +197,7 @@ def conv3d(signals, filters,
     Another way to define signals: (batch,  time, in channel, row, column)
     Another way to define filters: (out channel,time,in channel, row, column)
 
-    For the GPU, you can use this implementation or
-    :func:`conv3d_fft <theano.sandbox.cuda.fftconv.conv3d_fft>`.
+    For the GPU, use nnet.conv3d.
```

Comment:
Here nnet.conv3d is AbstractConv3d.  We can always add alternative implementations though.